modules/administration-guide/pages/upgrading-che-7-41-on-openshift.adoc

```diff
@@ -11,67 +11,85 @@ The workspace engine and authentication system used up to and including {prod-pr
+* The following command line tools are available:
+** `oc`
+** `curl`
+** `jq`
 * The host running the upgrade commands is running on Linux.
+* Optional:
+.. All changes from all workspaces have been commited and pushed to their Git remotes.
+.. All workspaces have been stopped to avoid UX degradation.
+.. The {prod-prev-short} data have been backed up. See {prod-docs-url-backup-recovery}.
 
 .Procedure
 
-. Optionally push back changes to Git repositories in all workspaces, stop all workspaces to avoid user experience degradation, and backup {prod-short} data. See {prod-docs-url-backup-recovery}.
+. Download xref:attachment$migration/1-prepare.sh[1-prepare.sh].
++
+`1-prepare.sh` is a migration script that shuts down {prod-prev-short} and {identity-provider}, fetches users data, and dumps the {prod-short} database.
+
+. Download xref:attachment$migration/2-migrate.sh[2-migrate.sh].
++
+`2-migrate.sh` is a migration script that fetches {prod-prev-short} {identity-provider} and database data, and repopulates the database with updated data.
+
+. Download xref:attachment$migration/3-subscribe.sh[3-subscribe.sh].
++
+`3-subscribe.sh` is a migration script that deletes {prod-prev-short} Operator and {identity-provider} resources, updates the CheCluster CR, and creates a new {prod-short} Operator subscription.
+
+. Download xref:attachment$migration/4-wait.sh[4-wait.sh].
++
+`4-wait.sh` is a migration script that waits until {prod-short} is ready, which can take more than 5 minutes.
 
 . Set the environment variables to use in the migration scripts:
 +
 [source,bash,subs="+attributes"]
 ----
-export INSTALLATION_NAMESPACE={prod-namespace} #<1>
+export INSTALLATION_NAMESPACE={prod-namespace} # <1>
 export PRODUCT_ID={prod-id}
 export PRODUCT_DEPLOYMENT_NAME={prod-deployment}
 export PRODUCT_OPERATOR_NAME={prod-operator}
 export PRODUCT_OLM_STABLE_CHANNEL={prod-stable-channel}
 export PRODUCT_OLM_CATALOG_SOURCE={prod-stable-channel-catalog-source}
 export PRODUCT_OLM_PACKAGE={prod-stable-channel-package}
-export PRE_MIGRATION_PRODUCT_OPERATOR_NAMESPACE={prod-namespace}  #<1>
+export PRE_MIGRATION_PRODUCT_OPERATOR_NAMESPACE={prod-namespace} # <2>
 export PRE_MIGRATION_PRODUCT_SHORT_ID={pre-migration-prod-id-short}
 export PRE_MIGRATION_PRODUCT_DEPLOYMENT_NAME={pre-migration-prod-deployment}
 export PRE_MIGRATION_PRODUCT_OPERATOR_NAME={pre-migration-prod-operator}
 export PRE_MIGRATION_PRODUCT_CHE_CLUSTER_CR_NAME={pre-migration-prod-checluster}
 export PRE_MIGRATION_PRODUCT_IDENTITY_PROVIDER_DEPLOYMENT_NAME={identity-provider-id}
 export PRE_MIGRATION_PRODUCT_SUBSCRIPTION_NAME={pre-migration-prod-subscription}
 ----
-<1> Adapt to the {orch-namespace} where {prod-short} is installed.
+<1> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
+<2> `{prod-namespace}` or another {orch-namespace} where {prod-prev-short} was previously installed.
```

Comment:
This is how they're rendered:
![image](https://user-images.githubusercontent.com/78564304/172851668-2be3a803-a4d1-4c17-bcc8-6e00718c2e92.png)

I'd like to avoid `The` there, and I'm aware of the styleguide guidance to start a sentence with some capitalized word, but in this case adding `The` makes it more confusing in my opinion.

It's either (a) `eclipse-che` or (b) _another namespace_.